core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -2436,6 +2437,32 @@ Scan *RelExpr::getLeftmostScanNode() const
   return (Scan *)result;
 }
 
+
+
+Join * RelExpr::getLeftJoinChild() const
+{
+  RelExpr *result = (RelExpr *)this;
+  
+  while(result)
+    {
+      if (result->getOperatorType() == REL_LEFT_JOIN) break;
+      result = result->child(0);
+    }
+  return (Join *)result;
+}
+
+RelSequence* RelExpr::getOlapChild() const
```

Comment:
getOlapChild() is the only new method corrently used. The LeftJoin access method isn't used for now. I am going to comment it out. But if I do, will make generic one.